src/urbit/lib/chess.hoon

```diff
@@ -853,73 +886,117 @@
           =([%black %king] from-piece)
       ==
     --
+  ::
+  ::  check logic
+  ::
+  ::  determine if the current player’s
+  ::  king is threatened
   ++  in-check
     ^-  ?
     (~(in-check with-board board) player-to-move)
+  ::
+  ::  determine if the current player’s
+  ::  king can’t move and is in check
   ++  in-checkmate
     ^-  ?
     &(in-check no-legal-moves)
+  ::
+  ::  determine if the current player’s
+  ::  king can’t move but is not in check
   ++  in-stalemate
     ^-  ?
     &(!in-check no-legal-moves)
-  ++  draw-claimable  ::  XX no threefold repetition yet
+  ::
+  ::   TODO: add threefold repetition draw
```

Comment:
`XX` is the Hoon style for `TODO` - I forgot about this during my first pass (for all 3 of you guys). Please remove the `TODO` comment, and keep the `XX` comment.